src/SFA.DAS.EmploymentCheck.Functions/Application/Services/EmployerAccount/EmployerAccountService.cs

```diff
@@ -56,135 +60,87 @@ public class EmployerAccountService
             _hostingEnvironment = hostingEnvironment;
             _azureClientCredentialHelper = azureClientCredentialHelper;
             _azureServiceTokenProvider = azureServiceTokenProvider;
+            _repository = repository;
         }
         #endregion Constructors
 
-        #region GetPayeSchemes
-
-        public async Task<ResourceList> GetPayeSchemes(Models.EmploymentCheck apprenticeEmploymentCheck)
+        public async Task<EmployerPayeSchemes> GetEmployerPayeSchemes(Models.EmploymentCheck employmentChecksBatch)
         {
-            var resourceList = await Get<ResourceList>(apprenticeEmploymentCheck);
-
-            return resourceList;
-        }
+            Guard.Against.Null(employmentChecksBatch, nameof(employmentChecksBatch));
 
-        #endregion GetPayeSchemes
+            var resourceList = await Get<ResourceList>(employmentChecksBatch);
+            if (!resourceList.GetType().Equals(typeof(ResourceList)))
+            {
+                // no PayeScheme found, return an empty EmployerPayeScheme (caller should check for empty EmployerPayeScheme
+                return new EmployerPayeSchemes(employmentChecksBatch.AccountId, new List<string>());
+            }
 
-        #region Get
+            return new EmployerPayeSchemes(employmentChecksBatch.AccountId, resourceList.Select(x => x.Id).ToList());
+        }
 
-        public async Task<TResponse> Get<TResponse>(Models.EmploymentCheck apprenticeEmploymentCheck)
+        #region GetEmployerPayeSchemes
+        public async Task<TResponse> Get<TResponse>(Models.EmploymentCheck employmentCheckBatch)
```

Comment:
bit lost not sure what's going on here, can this be broken down into smaller private methods ?

Reply:
Just renamed the apprenticeEmploymentCheck  to employmentCheckBatch to be consistent with the naming elsewhere sine the name of the table in the Dec release but was changed to EmploymentCheck.